### PR TITLE
renaming Messsage to Message

### PR DESCRIPTION
**Pull request by Coeur**
fixing a spelling mistake, deprecating old name

### CODE REVIEW

File: Mixpanel/MPDesignerEventBindingResponseMessage.m [hunk 1]
```diff
@@ -26,3 +26,7 @@ - (NSString *)status
 }
 
 @end
+
+@implementation MPDesignerEventBindingResponseMesssage
```
Comment (samgreen) [inline Mixpanel/MPDesignerEventBindingResponseMessage.m]: Looks like you missed one

File: Mixpanel/MPDesignerEventBindingRequestMessage.m [hunk 2]
```diff
@@ -88,3 +88,7 @@ - (NSOperation *)responseCommandWithConnection:(MPABTestDesignerConnection *)con
 }
 
 @end
+
+@implementation MPDesignerEventBindingRequestMesssage
```
Comment (samgreen) [inline Mixpanel/MPDesignerEventBindingRequestMessage.m]: This one is still wrong

**Comment by samgreen** [comment]
I do not believe you can mark a class as `__deprecated`, @alex-hofsteede can you chime in on this?

**Comment by samgreen** [comment]
I'm also unable to find any reference to marking a class with the deprecated attribute from the LLVM manual. http://clang.llvm.org/docs/AttributeReference.html

**Comment by Coeur** [comment]
@samgreen http://stackoverflow.com/questions/7408980/how-can-i-deprecate-an-entire-protocol and http://stackoverflow.com/questions/3908715/how-do-i-flag-a-method-as-deprecated-in-objective-c-2-0/12407525#12407525

**Comment by samgreen** [comment]
Just tried this in Xcode 6 and I don't get a warning using the deprecated class. Any idea why?

**Comment by Coeur** [comment]
Maybe not supported by Xcode. :/

**Pull request closed without merging**